ui/src/reducers/index.js

```diff
@@ -37,16 +37,18 @@ export default (state = initialState, action) => {
       var fetchedApps = [];
       action.result.forEach(app => {
         var temp = [];
-        temp[0] = app.appId;
```

Comment:
Will the appID appears in another place? If not,  wdyt over keep both infos?

Reply:
Not sure what the question is here. This is managing the state of the apps page so it would only appear once on that page. We'll probably have a different object for the state of the detailed apps ui which will get populated on the get request for the app/{id}

Reply:
Ah I understand the question now. This array is what's used to render the table and we use the appName in the table and not the appId as we discussed in the chat.

Reply:
In the lines below you can see that I'm storing the full data object returned by the request. See `data: action.result`

Reply:
IMHO both info is important to the users since the appName is the name given for them to create the app and the app_id is the data that the mobile device via SDK should send in order to make it works.